src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -297,7 +350,7 @@ public void run() {
                 // @checkstyle MagicNumber (1 line)
                 final byte[] buffer = new byte[4096];
                 for (int count = input.read(buffer); count != -1;
-                    count = input.read(buffer)) {
+                     count = input.read(buffer)) {
```

Comment:
@dmzaytsev Thanks for link.
Done with issue: https://github.com/teamed/xdsd/issues/27